src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react'
+
+import cn from 'classnames'
+
+import styles from './ProgressBar.module.css'
+import { ProgressBarProps } from './types'
+
+const clamp = (value: number, min: number, max: number): number =>
+  Math.min(Math.max(value, min), max)
+
+export const ProgressBar: React.FC<ProgressBarProps> = ({
+  className,
+  sliderClassName,
+  sliderBarClassName,
+  min = 0,
+  max = 1,
+  value,
```

Comment:
how will this play with BN going forward? would you convert to a number? you might actually lose precision since BN can represent bigger than max number

Reply:
I was thinking this would be used with normal numbers and the TokenValueSlider would be the BN one for token balances. Can update if this needs to work with BN though

Reply:
Updated to support `number | BN` 👍